extensions/Bookalope/css/bookalope.css

```diff
@@ -1,185 +1,542 @@
+	box-sizing: content-box;
+	height: 0;
+	overflow: visible
 }
 
-.hostFont {
+pre {
+	font-family: monospace, monospace;
+	font-size: 1em
 }
 
-.hostBgd {
+a {
+	background-color: transparent
 }
 
-.hostBgdLight {
+abbr[title] {
+	border-bottom: none;
+	text-decoration: underline;
+	-webkit-text-decoration: underline dotted;
+	text-decoration: underline dotted
 }
 
-.hostBgdDark {
+b, strong {
+	font-weight: bolder
 }
 
-.hostElt {
+code, kbd, samp {
+	font-family: monospace, monospace;
+	font-size: 1em
 }
 
-/* Panel styling */
+small {
+	font-size: 80%
+}
 
-@font-face {
-  font-family: "Noto Sans";
-  src: url("../font/NotoSans-Regular.ttf");
+sub, sup {
+	font-size: 75%;
+	line-height: 0;
+	position: relative;
+	vertical-align: baseline
+}
+
+sub {
+	bottom: -0.25em
+}
+
+sup {
+	top: -0.5em
+}
+
+img {
+	border-style: none
+}
+
+button, input, optgroup, select, textarea {
+	font-family: inherit;
+	font-size: 100%;
+	line-height: 1.15;
+	margin: 0
+}
+
+button, input {
+	overflow: visible
+}
+
+button, select {
+	text-transform: none
+}
+
+button, [type="button"], [type="reset"], [type="submit"] {
+	-webkit-appearance: button
+}
+
+button::-moz-focus-inner, [type="button"]::-moz-focus-inner, [type="reset"]::-moz-focus-inner, [type="submit"]::-moz-focus-inner {
+	border-style: none;
+	padding: 0
+}
+
+button:-moz-focusring, [type="button"]:-moz-focusring, [type="reset"]:-moz-focusring, [type="submit"]:-moz-focusring {
+	outline: 1px dotted ButtonText
+}
+
+fieldset {
+	padding: 0.35em 0.75em 0.625em
+}
+
+legend {
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+	color: inherit;
+	display: table;
+	max-width: 100%;
+	padding: 0;
+	white-space: normal
+}
+
+progress {
+	vertical-align: baseline
+}
+
+textarea {
+	overflow: auto
+}
+
+[type="checkbox"], [type="radio"] {
+	-webkit-box-sizing: border-box;
+	box-sizing: border-box;
+	padding: 0
+}
+
+[type="number"]::-webkit-inner-spin-button, [type="number"]::-webkit-outer-spin-button {
+	height: auto
+}
+
+[type="search"] {
+	-webkit-appearance: textfield;
+	outline-offset: -2px
+}
+
+[type="search"]::-webkit-search-decoration {
+	-webkit-appearance: none
+}
+
+::-webkit-file-upload-button {
+	-webkit-appearance: button;
+	font: inherit
+}
+
+details {
+	display: block
+}
+
+summary {
+	display: list-item
+}
+
+template {
+	display: none
+}
+
+[hidden] {
+	display: none
 }
+
+.hidden {
+	display: none !important
+}
+
 @font-face {
-  font-family: "Noto Serif";
-  src: url("../font/NotoSerif-Regular.ttf");
+	font-family: 'Noto Sans';
+	src: url("../font/NotoSans-Regular.ttf") format("truetype");
+	font-weight: normal;
+	font-style: normal
 }
 
 body {
-  color: #d6d6d6;
-  background-color: #494949;
-  font: normal 12px "Noto Sans", Helvetica, Arial, sans-serif;
-  padding: 10px;
- /* Enable for browser testing.
-  width: 400px;
-  height: 290px;
- */
+	font-family: adobe-clean, "Noto Sans", "Source Sans Pro", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif
```

Comment:
I’ve checked with a few people in the [Adobe #cep](https://adobedevs.slack.com/archives/C1FKLQ63F/p1560150722075000) channel: the official fonts for CEP extensions are
```css
adobe-clean, "Source Sans Pro",  -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif
```
I think we can remove Noto completely.

Reply:
Removed "Noto Sans" font.

Reply:
Do you think we should add the font files for Source Sand Pro?

Reply:
Added font files for Source Sand Pro.

Reply:
Commit https://github.com/jenstroeger/Bookalope-CEP/pull/1/commits/9280c2002c6e747225335156cf44f25a8874a50d.